src/main/java/lotto/Lotto.java

```diff
@@ -0,0 +1,69 @@
+package lotto;
+
+import java.util.List;
+import java.util.Scanner;
+
+import lotto.dto.LottoResultAggregation;
+import lotto.model.LottoMatchResults;
+import lotto.model.LottoNumbers;
+import lotto.model.LottoReward;
+import lotto.model.WinningNumber;
+import lotto.view.GeneratedLottoNumberResponseView;
+import lotto.view.LottoResultAggregationResponseView;
+import lotto.view.LottoYieldResponseView;
+
+public class Lotto {
+    private static final Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        int paymentAmount = getPaymentAmount();
+        int lottoQuantity = LottoVendingMachine.calculate(paymentAmount);
+        showBuyResult(lottoQuantity);
+
+        // create Lotter numbers
+        LottoNumbers lottoNumbers = LottoVendingMachine.buy(lottoQuantity);
+
+        showGeneratedLottoNumbers(lottoNumbers.getLottoNumbers());
+
+        WinningNumber winningNumber = getWinningNumber();
+
+        System.out.println("당첨통계");
+        System.out.println("------------------");
+        LottoMatchResults lottoMatchResults = new LottoMatchResults(lottoNumbers.guess(winningNumber));
+        LottoResultAggregation lottoResultAggregation = new LottoResultAggregation(lottoMatchResults.rewardableAggregate());
+
+        showLottoResultAggregation(lottoResultAggregation);
+        showLottoYield(paymentAmount, lottoMatchResults.sumRewardAmount());
+    }
+
+    private static int getPaymentAmount() {
+        System.out.println("구입금액을 입력해주세요.");
+
+        int paymentAmount = scanner.nextInt();
+        scanner.nextLine();
+        return paymentAmount;
+    }
+
+    private static void showBuyResult(int lottoQuantity) {
+        System.out.println(String.format("%d개를 구매했습니다.", lottoQuantity));
+    }
+
+    private static void showGeneratedLottoNumbers(List<List<Integer>> lottoNumbers) {
+        System.out.println(new GeneratedLottoNumberResponseView(lottoNumbers).toView());
```

Comment:
별도의 변환 처리없이 `List<Integer>`를 그대로 출력해도 되지 않을까요?
```suggestion
        for (List<Integer> lottoNumber : lottoNumbers) {
            System.out.println(lottoNumber);
        }
```